src/databricks/labs/remorph/helpers/db_sql.py

```diff
@@ -16,13 +16,16 @@
 
 
 def get_sql_backend(ws: WorkspaceClient, config: MorphConfig) -> SqlBackend:
-    sdk_config = ws.config
-    warehouse_id = isinstance(sdk_config.warehouse_id, str) and sdk_config.warehouse_id
+    sdk_config = config.sdk_config
+    warehouse_id = sdk_config.get("warehouse_id", None) if sdk_config else None
```

Comment:
Should we fallback to ws.config.warehouse_id?

Reply:
I m expecting user makes that choice for us at earlier step only.

Reply:
Refer to my previous comment